src/yowsup_if.js

```diff
@@ -836,35 +837,7 @@ CoSeMe.namespace('yowsup.readerThread', (function() {
 
   var alive = false;
 
-  var _onPing;
-  var _ping;
-  var _autoPong;
-
   return {
-    set onPing(aFun) {
```

Comment:
you are removing this setter, but you are still using the function (if it exists) in the first line. I don't think this is good code